lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -918,15 +939,49 @@ def rotate_axes(xs, ys, zs, zdir):
         return xs, ys, zs
 
 
-def _zalpha(colors, zs):
+def _zalpha(colors, zs, _dscl=None):
```

Comment:
What is `_dscl`? No need to be stingy with letters in variable names.

Reply:
How about "_data_scale", then?